### PR TITLE
Update Sony - PlayStation (Redump).json

### PR DESCRIPTION
**Pull request by rufotheone-fr**
Sorry for the absence, I was kinda busy ^^;
GG again for Retool 2.0, I'm playing with it ATM.

### CODE REVIEW

**Comment by unexpectedpanda** [comment]
Good to see you! Welcome back :)

File: clonelists/Sony - PlayStation (Redump).json [hunk 1]
```diff
@@ -3254,6 +3254,12 @@
 				{"searchTerm": "Kagero - Kokumeikan Shinshou"}
 			]
 		},
+		{
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation (Redump).json:3257]: What would you like to achieve with this change? With only one title in the list, this won't do anything.

**Comment by rufotheone-fr** [comment]
Do I have to create a group for Kamen Rider V3 too?

**Comment by unexpectedpanda** [comment]
> Do I have to create a group for Kamen Rider V3 too?

Only if the title called "Kamen Rider" is different.

See the PlayStation 2 overrides section for how to achieve this.

**Comment by rufotheone-fr** [comment]
Thank you UP for the explanations and your GREAT patience u_u

**Comment by rufotheone-fr** [comment]
BTW, we have a strange case with Ghost in the Shell...
The game was dubbed in french and german for those countries but texts weren't translated. Redump don't add the countries flag when the text isn't translated so, even if I choose to have games in french and fall back to US when it's not translated, GitS (France) is skipped for GitS (USA). Do we have to edit the metadata?

**Comment by unexpectedpanda** [comment]
> Thank you UP for the explanations and your GREAT patience u_u

Thanks for your contributions! I'm really stoked each time I see one :)

This one is good too, because it highlights where I can make things easier for people to contribute. In this instance I've realized that I can probably get away with _not_ changing the group names in clone lists -- which would make things much less confusing for everyone. I'll look into it.

> BTW, we have a strange case with Ghost in the Shell...
The game was dubbed in french and german for those countries but texts weren't translated. Redump don't add the countries flag when the text isn't translated so, even if I choose to have games in french and fall back to US when it's not translated, GitS (France) is skipped for GitS (USA). Do we have to edit the metadata?

So, repeating what you're saying to make sure I understand correctly: _Ghost in the Shell (France)_ is marked by Redump as being in English, but that's only because text in the game is in English, while the voicework is dubbed in French or German (or at least, those dubs are available).

The only scenario where you should still get the USA variation chosen over the French one (even if you prefer French over English as a language) is if USA is higher in your region order than France. If you've set up Retool differently and are still seeing the USA version selected, I'd be interested to know your settings.

Assuming you've configured Retool that way, this is a tough problem. We can't edit the metadata to change the languages, as it's overwritten each time I scrape Redump's site. I've also historically resisted turning Retool into a language database, preferring to fix the issue at the source, but the problem here is the language standard both Redump and No-Intro follow. Both fail to differentiate between text used in the game, subtitles, and voicework for languages. Both don't indicate what the default language of the title is, assuming there is one (and this can be tricky, as I believe some can switch the default language depending on the region the console is in).

Supporting this sort of functionality in Retool isn't a small thing, and requires a lot of verification work. I'll think about it, but at the minimum all the data would have to be community-supplied. For now, unfortunately we're stuck with this suboptimal situation.

**Comment by rufotheone-fr** [comment]
Exactly! I've setted USA in a higher order than France but placed french language on top so I can get the best of the two worlds (NTSC games that have FR in their languages, NTSC games when games are in english only and PAL games when they're the only ones translated in FR).

For GitS, the way that game works is sick... GitS (France)/(Germany) has french/german voiceover without subtitles (and there's A LOT of cutscenes and briefings) but the texts are in EN.

**Comment by unexpectedpanda** [comment]
Totally makes sense :)

I don't think I can work around it the way things are built currently, and there's a lot of complexity and work in supporting the different ways languages are presented. The only way forward at the moment is to add your own system override to remove the GiTS titles you don't want.

I'll mark it down as a problem title though, and should I get around to looking at a larger language project, I'll make sure it's considered.

**Pull request merged**